__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require('supertest');
+const app = require('../app')
+
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => seed(testData));
+
+
+describe('GET /api/topics', ()=>{
+    test('200: responds with an array of topic objects, each of which should have the following properties: slug and description', ()=>{
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((res)=>{
+                console.log(res.body, '<<<res.body')
+                expect(res.body).toBeInstanceOf(Object);
+                expect(res.body).not.toBeInstanceOf(Array);
+                expect(res.body).toEqual({
+                    topics: [
+                      { slug: 'mitch', description: 'The man, the Mitch, the legend' },
+                      { slug: 'cats', description: 'Not dogs' },
+                      { slug: 'paper', description: 'what books are made of' }
+                    ]
+                  })
+            })
+    })
+})
```

Comment:
you want to have the objects here not the whole array, res.body.topics will help you. Then you can test for what you are getting in each object.

Reply:
tested each object by itself and added another test if path does not exist